### DISCUSSION TITLE
How can I become tw css's contributer?

### DISCUSSION THREAD

**Discussion by qingyggg** [tailwindlabs/tailwindcss#8407] [Help] [answered]
How can I become tw css's contributer?

**Comment by Erin69** [comment, marked as answer]
There is no one way to contribute to an open source project, and to be involved often means more than code slinging. In this article, though, we’re going to focus on the nitty-gritty of contributing a pull request (PR) to someone else’s project on GitHub.
[MyLowesLife](https://www.myloweslife.bid/)
